### ISSUE TITLE
windows not installing static lib

### ISSUE THREAD

**Issue by djsutherland** [conda-forge/shogun-cpp-feedstock#14]
Files in the current Windows package:

- Library\include\shogun\*
- Library\share\shogun\examples\libshogun\minimal.exe
- Library\lib\shogun.lib

@vigsterkr, should it be installing the static library too?

**Comment by vigsterkr** [comment]
yeah static lib is something we want... i'm wondering why the dll is not being installed :(

**Comment by djsutherland** [comment]
Really hard to fiddle with without a Windows machine and when the CI takes an hour to get to the point of installation....

**Comment by vigsterkr** [comment]
yep indeed... lemme check this error:
```
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppCommon.targets(171,5): error MSB6001: Invalid command line switch for "cmd.exe". The path is not of a legal form. [C:\bld\shogun_1511965094444\work\shogun-shogun_6.1.2\build\src\interfaces
```

**Comment by vigsterkr** [comment]
so there was somebody who already built the java interface on windows:
https://github.com/shogun-toolbox/shogun/issues/3948

i dont think the person used conda but see the build flags for msbuild:
```
msbuild shogun.sln /p:Configuration=Release /p:Platform=x64 /p:WarningLevel=0 /v:m /m
```

**Comment by vigsterkr** [comment]
@dougalsutherland ^

**Comment by djsutherland** [comment]
So none of the previous `msbuild` lines, just `shogun.sln` right away? Will try. (And add `/v:m /m`, who knows what those do....)

**Comment by vigsterkr** [comment]
@dougalsutherland dont ask about the flags.... apparently they work :)

**Comment by vigsterkr** [comment]
ok this is fixed by #18
